src/OrchardCore/OrchardCore.Mvc.Core/Routing/RoutingHelper.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Mvc.Controllers;
+
+namespace OrchardCore.Mvc.Routing;
+
+public static class RoutingHelper
```

Comment:
It's best practice that [Static holder types should be Static or NotInheritable](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1052).

Reply:
What about adding `sealed` then?

Reply:
The `static` modifier implicitly makes the class `sealed` too:

![image](https://github.com/OrchardCMS/OrchardCore/assets/4007293/fe1e4281-4fe8-430a-9e13-ef15a3009d2d)

Also it's not possible to mark a `static` class explicitly `sealed`:

![image](https://github.com/OrchardCMS/OrchardCore/assets/4007293/7a09b692-076c-42a7-b1ae-3d52ac9b5cbd)

Reply:
sealed without static.

```
public sealed class RoutingHelper
```

Reply:
We add the `static` modifier to the class because helpers should only contain static methods anyway. With `sealed` you can still add instance methods or instantiate the type using `new RoutingHelper();` even though that makes no sense. In the Microsoft documentation I linked previously:
![image](https://github.com/OrchardCMS/OrchardCore/assets/4007293/2bfad1a7-ae82-4576-aa76-efa9b0766cf6)